.github/workflows/ci_pr.yml

```diff
@@ -88,6 +88,8 @@ jobs:
       matrix:
         include:
           - python-version: "3.5"
+            # workaround for issue "[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:728)" on Python 3.5
+            pip_trusted_host: "pypi.python.org pypi.org files.pythonhosted.org"
```

Comment:
good find; could you add https://github.com/actions/setup-python/issues/866 to the comment?